src/Query/EloquentQueryBuilder.php

```diff
@@ -40,6 +43,8 @@ public function __call($method, $args)
     {
         $response = $this->builder->$method(...$args);
 
+        $this->applyScope($method);
```

Comment:
Previous to this change, you could add scopes to your Eloquent model that'd be used when doing `User::query()`. 

Now, with this PR, an error is thrown because it can't find a Statamic query scope with that name.

For example: 

```php
use Statamic\Facades\User;

User::query()->duncan()->get();
```

```php
public function scopeDuncan($query)
{
    $query->where('email', 'duncan@statamic.com');
}
```

Reply:
That's probably due to this line here:
https://github.com/statamic/cms/blob/002e765cf0a570d9a93ba741bc2497623a0f3edb/src/Query/Scopes/AppliesScopes.php#L12-L14

Maybe we can add a line before that to return early if a scope method is found on the model.

Reply:
Yeah, maybe we can do something before the `applyScope` to workaround it.

Reply:
I've sorted it out - the `EloquentQueryBuilder` will check if a Statamic query scope exists before falling back to any scopes defined on Eloquent models.